ftw/simplelayout/browser/resources/main.js

```diff
@@ -148,6 +148,12 @@
         }
       });
 
+      var initializeColorbox = function() {
+        if($(".colorboxLink").length > 0) {
+          ftwColorboxInitialize();
```

Comment:
Make sure the `ftwColorboxInitialize` is available. In case of plone5, or old colorbox version this method does not exist.